File: app/src/main/java/protect/card_locker/CatimaBarcodeWithValue.java
package protect.card_locker;

public class CatimaBarcodeWithValue {
    private final CatimaBarcode mCatimaBarcode;
    private final String mValue;

    public CatimaBarcodeWithValue(CatimaBarcode catimaBarcode, String value) {
        mCatimaBarcode = catimaBarcode;
        mValue = value;
    }

    public CatimaBarcode catimaBarcode() {
        return mCatimaBarcode;
    }

    public String value() {
        return mValue;
    }
}

File: app/src/main/java/protect/card_locker/CatimaBarcodeWithValue.kt
package protect.card_locker

class CatimaBarcodeWithValue(
    private val mCatimaBarcode: CatimaBarcode?,
    private val mValue: String?
) {
    fun catimaBarcode(): CatimaBarcode? {
        return mCatimaBarcode
    }

    fun value(): String? {
        return mValue
    }
}
